pydis_site/apps/content/resources/guides/python-guides/keeping-tokens-safe.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Keeping Discord Bot Tokens Safe
+description: How to keep your bot tokens safe and safety measures you can take.
+---
+It's **very** important to keep a bot token safe, primarily because anyone who has the bot token can do whatever they want with the bot -- such as destroying servers your bot has been added to and getting your bot banned from the API.
+
+# How to Avoid Leaking your Token
+To help prevent leaking your token, you should ensure that you don't upload it to an open source program/website, such as replit and github, as they show your code publicly. The best practice for storing tokens is generally utilising .env files ([click here](https://vcokltfre.dev/tips/tokens/.) for more information on storing tokens safely)
```

Comment:
It may be best for us to use a different website than this one, due to the website's creator no longer being a part of this org.

Along with that, the we should probably wrap `.env` in a minature code block, shown below.

```suggestion
To help prevent leaking your token, you should ensure that you don't upload it to an open source program/website, such as replit and github, as they show your code publicly. The best practice for storing tokens is generally utilising `.env` files to export [environment variables](https://en.wikipedia.org/wiki/Environment_variable#Assignment:_Unix).
```

Reply:
I disagree, it was there because it's a good resource, and it still is, I don't think who created it is that relevant.

Reply:
Very well.

Reply:
The second point still stands, @Shom770 can you use codelines for `.env`?

Reply:
> The second point still stands, @Shom770 can you use codelines for `.env`?
What's codelines? I'm a bit lost here.

Reply:
@Shom770 Use backticks ` around .env on line 8